compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/temporary_alloc_bufferization.mlir

```diff
@@ -86,3 +76,34 @@ func.func @temp_buffer() {
   }
   return
 }
+
+// -----
+
+// CHECK-LABEL: @temp_buffer_2
+//   CHECK-DAG:   %[[C0:.*]] = arith.constant 0 : index
+//   CHECK-DAG:   %[[C1:.*]] = arith.constant 1 : index
+//   CHECK-DAG:   %[[TILE_0:.*]] = amdaie.tile(%[[C0]], %[[C0]])
+//   CHECK-DAG:   %[[BUFFER_0:.*]] = amdaie.buffer(%[[TILE_0]]) : memref<4xi32>
+//   CHECK-DAG:   %[[TILE_1:.*]] = amdaie.tile(%[[C0]], %[[C1]])
+//   CHECK-DAG:   %[[BUFFER_1:.*]] = amdaie.buffer(%[[TILE_1]]) : memref<4xi32>
+//       CHECK:   amdaie.core(%[[TILE_0]]
+//       CHECK:     %[[CAST:.*]] = memref.reinterpret_cast %[[BUFFER_0]]
+//       CHECK:   amdaie.core(%[[TILE_1]]
+//       CHECK:     %[[CAST:.*]] = memref.reinterpret_cast %[[BUFFER_1]]
```

Comment:
I would prefer for style consistency that we don't indent the `CHECK` and place them one space after `//` as we do in the rest of the codebase (except for this file apparently :P).

Reply:
Noted. :P